### PR TITLE
Added link to the community page in the footer

### LINKED ISSUE #10473: Add link to community page on all websites
A link to https://bokeh.org/community/ needs to be added to the footers:
- [ ] https://docs.bokeh.org/en/latest/
- [ ] https://demo.bokeh.org/

### PR DESCRIPTION
**Pull request by AdityaJ7**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #10473

### CODE REVIEW

**Comment by AdityaJ7** [comment]
Hey, can someone tell me how do I fix this failing check?

**Comment by pavithraes** [comment]
@AdityaJ7 The checks are failing because:
```
E       AssertionError: Code quality issues:
E         File contains tabs instead of spaces: sphinx/source/bokeh/footer.html, line 45.
E       assert 1 == 0
E        +  where 1 = len(['File contains tabs instead of spaces: sphinx/source/bokeh/footer.html, line 45.'])
```
For consistency, spaces are used for indentation instead of tabs in the codebase. You need to change the tab on line 45 to spaces. :)

File: sphinx/source/bokeh/footer.html [hunk 1]
```diff
@@ -42,6 +42,9 @@ <h6>Hands On</h6>
             <li class="nav-item">
               <a class="nav-link" href="https://demo.bokeh.org">Demos</a>
             </li>
+	    <li class="nav-item">
```
Comment (pavithraes) [inline sphinx/source/bokeh/footer.html:45]: Besides the issue with tabs, we can also fix the alignment here to be consistent with the previous tags.

**Review by pavithraes — CHANGES_REQUESTED** [review]

File: sphinx/source/bokeh/footer.html (hunk 1 above)
Comment (AdityaJ7) [reply sphinx/source/bokeh/footer.html]: Made the changes, I must have messed up when I tried fixing this spacing, thanks again @pavithraes

**Review by pavithraes — APPROVED** [review]

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @AdityaJ7 !

**Comment by AdityaJ7** [comment]
> Thanks @AdityaJ7 !

Happy to contribute

**Pull request merged**